src/components/gallery/HdGalleryMedia.vue

```diff
@@ -101,6 +105,11 @@ export default {
       z-index: 1;
     }
```

Comment:
Here you want to have 
```css
    &-thumbnail,
    &-picture,
    &-video {
      position: absolute;
      top: 0;
      right: 0;
      bottom: 0;
      left: 0;
    }
    &-thumbnail {
        z-index: 1;
    }
    
    &-picture,
    &-video {
        z-index: 2;
    }
```

So picture and video are accessible (a.k.a. the video is clickable)

Reply:
yeah, this is also what I thought, but then I saw `@load="hideThumbnail"` on `img` and `iframe` element.
So basically, thumbnail is on top until `img` or `iframe` gets loaded and will hide the thumbnail automatically.
@drapisarda

Reply:
Those Gifs are captured from the storybook. Can you share a gif here?

Reply:
Yeah, apparently it is not working for me too. The video loads but it can't be played.
![HdGalleryVideo](https://user-images.githubusercontent.com/7534298/178906959-e6648a67-4bf3-4101-be47-7af38f6eb78b.gif)

Reply:
Thank you @aym3nb , now I see.